cypress/integration/page.spec.js

```diff
@@ -455,7 +484,7 @@
     describe('on posts loading error', () => {
       beforeEach(() => {
         page.mockUsers();
-        page.mockError('**/posts?userId=1', 'user1PostsRequest')
+        page.mockError('**/posts?userId=1', 'user1PostsRequest');
```

Comment:
The `page.mockError` call should be followed by `.as('user1PostsRequest')` to ensure the request can be waited on or asserted later in the test.